### DISCUSSION TITLE
Uploading the pem files to enable SSL

### DISCUSSION THREAD

**Discussion by NgweBecky** [camicroscope/caMicroscope#687] [Q&A]
@birm please kindly brief me on how I can create the private key and certificate pem files to upload on the ca-back service. I encounter the error attached in the screenshot for the ca-load service, please any suggestion? I noticed the directory mentioned in the readme.md file which is "/root/src/ssl" does not exist, I wonder if this is due to the ca-load error.
![ca-load error](https://github.com/camicroscope/caMicroscope/assets/54197600/42df705d-c4ad-4814-b271-36efe287bcd1)

**Comment by birm** [comment]
You may already seen [this](https://github.com/camicroscope/camicroscope.github.io) in the readme, but it is pretty sparse for detail. You should be able to use https://github.com/camicroscope/Distro/blob/master/keys/make_keys.sh within the ./keys dir within distro, or add a compatible key.
Ca-load _should_ not have anything to do with ssl keys; that issue was related to an openslide library object that python wouldn't find. It's hopefully resolved if you rebuild without cache.
